packages/zoe/src/contracts/automaticRefund.js

```diff
@@ -1,34 +1,32 @@
 // @ts-check
-import rawHarden from '@agoric/harden';
+import harden from '@agoric/harden';
 
-// TODO: Until we have a version of harden that exports its type.
-const harden = /** @type {<T>(x: T) => T} */ (rawHarden);
+// Eventually will be importable from '@agoric/zoe-contract-support'
+import { makeZoeHelpers } from '../contractSupport';
```

Comment:
@michaelfig Do we still need this ceremony around harden? I thought I saw that you'd published a new harden package with types so we didn't need this here anymore. True?

Reply:
> I don't think we should be changing this in this PR, since it is unrelated.

Agreed.  We need to update all the packages to use the new version of harden in their `package.json`.

Reply:
All occurrences reverted